jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -133,11 +133,14 @@ public final class System {
 	private static String fileEncoding;
 	private static String osEncoding;
 	private static String defaultTmpDir;
+	private static String zOSAutoConvert;
```

Comment:
This field and `sysPropID_zOSAutoConvert` should be conditional. Suggest reorganizing and putting the identifiers and values in pairs:
```java
	/*[IF JAVA_SPEC_VERSION >= 17]*/
	private static final int sysPropID_OSVersion = 0;
	private static final String sysPropOSVersion;
	/*[ENDIF] JAVA_SPEC_VERSION >= 17 */

	private static final int sysPropID_PlatformEncoding = 1;
	private static String platformEncoding;

	private static final int sysPropID_FileEncoding = 2;
	private static String fileEncoding;

	private static final int sysPropID_OSEncoding = 3;
	private static String osEncoding;

	private static final int sysPropID_DefaultTmpDir = 4;
	private static String defaultTmpDir;

	/*[IF (JAVA_SPEC_VERSION >= 21) & (PLATFORM-mz31 | PLATFORM-mz64)]*/
	private static final int sysPropID_zOSAutoConvert = 5;
	private static String zOSAutoConvert;
	/*[ENDIF] (JAVA_SPEC_VERSION >= 21) & (PLATFORM-mz31 | PLATFORM-mz64) */
```

Reply:
Updated under: [9c68f5c](https://github.com/eclipse-openj9/openj9/pull/19772/commits/9c68f5caec292c7502c6c26dfdfaa201282b060a)